packages/calcite-components/src/components/tree-item/tree-item.tsx

```diff
@@ -227,17 +231,22 @@ export class TreeItem implements ConditionalSlotComponent, InteractiveComponent
 
     const checkbox =
       this.selectionMode === "ancestors" ? (
-        <label class={CSS.checkboxLabel} key="checkbox-label">
-          <calcite-checkbox
-            checked={this.selected}
-            class={CSS.checkbox}
-            data-test-id="checkbox"
-            indeterminate={this.hasChildren && this.indeterminate}
-            scale={this.scale}
-            tabIndex={-1}
-          />
-          {defaultSlotNode}
-        </label>
+        <div class={CSS.checkboxContainer}>
+          <div class={CSS.checkbox} tabIndex={-1}>
```

Comment:
Does this need a tabindex of 1? Are we ever focusing this element?

Reply:
It looks like it only gets focused on click, should I remove tabIndex?

Reply:
I think so, whats the reasoning for focusing it?

Reply:
We apply this css block when focusing the checkbox:

`.checkbox:focus {
  @apply outline-none;
  color: var(--calcite-color-brand);
}`

Reply:
But this focus would only happen when clicking on that element.

Could we apply this style to the checkbox when the tree-item host is focused? Then it would be consistent.

Reply:
Something like

```css
         :host(:focus:not([disabled])) .checkbox{ @apply outline-none; color: var(--calcite-color-brand); } 
```

Might also be nice to have on `hover` and `active` as well.

Reply:
I updated this to apply the focused style when tree-item host is focused. 
Regarding the `hover` state, design previously reviewed this and they don't want any styles applied when items are hovered.

Reply:
I think this one still needs to be addressed. 

1. Do we need/want the checkbox node to be focusable? (probably not).
2. Do we need hover/active/focus states on the host that style this? (maybe?)

Reply:
@aPreciado88 styles look good, but doesn't the `tabindex` of `-1` need to be removed?

Reply:
@driskull Design reviewed the latest chromatic builds, and asked to remove the `focused` checkbox state. I pushed an update for that.